ONSdigital/dp-frontend-renderer
# Feature/add dataset template

### What

Migrated dataset page template from babbage to handle "dataset" page types in the renderer.
Added routing for new dataset page template.

### How to review

- Look at the code. 
- Pull the code into your local environment with the corresponding dp-frontend-dataset-controller and dp-frontend-router branches. 
- Set 'DATASET_ENABLED' flag to true in environment config.
- Build and run the web stack. 
- Navigate to an Old Workspace dataset versions page locally. e.g. /economy/inflationandpriceindices/datasets/consumerpriceinflation/current. 
- See no changes in the page and it works exactly as before pulling the new code.
- 

### Who can review

Anyone.
